src/Files.App/UserControls/SideBar/SideBarControls.xaml

```diff
@@ -106,20 +107,25 @@
 								x:Name="IconPresenter"
 								Width="16"
 								Height="16"
+								Margin="11,0,0,0"
```

Comment:
Is an odd margin required?

Reply:
I'll revert because this is out of scope.
11 is a mixed up margin of 3px of selection indicator and 8px spacing.

Reply:
I should've use ColumnDefinition in the first place tho

Reply:
Reverted